src/Nordea/Resources/Icons.elm

```diff
@@ -226,6 +211,34 @@ chevronDownFilled attrs =
         ]
 
 
+chevronDownFilledSmall : List (Attribute msg) -> Html msg
+chevronDownFilledSmall attrs =
```

Comment:
Is this a different icon, and not just a smaller version of the same icon?

Reply:
It is a smaller version of the normal one, but just shrinking the original doesn't work, because the ratio between background and the arrow itself is different. So to keep the size of the arrow, while reducing the size of the whole box, we needed to create a new icon☺️ Could be a method to do this that I'm not aware of though? Just scaling it down at least did not work